apps/dashboard/app/apps/ood_app.rb

```diff
@@ -264,7 +264,14 @@ def run_setup_production
           #
           # This makes the execution of the setup-production script use the same ruby versions
           # that Passenger uses when launching the app.
-          output, status = Open3.capture2e({'PATH' => path.join('bin').to_s + ':'+ ENV['PATH']}, 'bundle','exec', setup)
+          if File.exist?('./Gemfile')
+            cmd = 'bundle'
+            args = ['exec', setup]
+          else
+            cmd = setup
+            args = []
+          end
+          output, status = Open3.capture2e({'PATH' => path.join('bin').to_s + ':'+ ENV['PATH']}, cmd, args)
```

Comment:
We need to use the single `splat` operator on `args` so it'd become `*args`.  Otherwise I get a type error about no implicit conversion from Array to String.


![image](https://github.com/OSC/ondemand/assets/4874123/3276c5b5-7a8f-474b-a38d-c59b594b6dc2)